setup.py

```diff
@@ -27,62 +27,65 @@ def read(*filenames, **kwargs):
 
 
 root = os.path.dirname(os.path.realpath(__file__))
-version_module = imp.load_source(
-    "version", os.path.join(root, "nengo_ocl", "version.py")
-)
-testing = "test" in sys.argv or "pytest" in sys.argv
-
-# Don't mess with added options if any are passed
-sysargs_overridden = False
+version = runpy.run_path(os.path.join(root, "nengo_ocl", "version.py"))["version"]
 
-if testing and "--addopts" not in sys.argv:
-    # Enable nengo tests by default
-    old_sysargs = sys.argv[:]
-    sys.argv[:] = old_sysargs + ["--addopts", "--pyargs nengo"]
-    sysargs_overridden = True
+install_req = [
+    "nengo>=3.0.0,<3.1.0",
+    "mako",
+    "pyopencl",
+]
+docs_req = [
+    "matplotlib>=1.4",
+    "nbsphinx",
+    "nengo-sphinx-theme>=1.2.0",
+    "numpydoc>=0.6.0",
+    "sphinx>=1.8",
+]
+optional_req = []
+tests_req = [
+    "matplotlib>=1.4",
+    "pytest>=3.6.0",
+    "pytest-allclose>=1.0.0",
+    "pytest-plt>=1.0.0",
+    "pytest-rng>=1.0.0",
+]
 
 setup(
     name="nengo-ocl",
-    version=version_module.version,
+    version=version,
     author="Applied Brain Research",
     author_email="info@appliedbrainresearch.com",
     packages=find_packages(),
-    scripts=[],
-    data_files=[],
-    url="https://github.com/nengo/nengo-ocl",
+    url="https://www.nengo.ai/nengo-ocl",
```

Comment:
This should be `labs.nengo.ai`, but the `www` part is hard-coded into nengo-bones right now. I wonder though, would it be possible to set up a redirect from `www.nengo.ai/nengo-ocl` to `labs.nengo.ai/nengo-ocl`? Might be a useful thing to have in general, for anyone typing in the address wrong.

Reply:
I'm not 100% sure since there used to be a repo there, but we can try adding the redirect on `nengo.github.io` and see if it works. If it doesn't (though I think it should) then we'd have to recreate the `nengo/nengo-ocl` repo just as a way to redirect to the new docs.

Reply:
I agree, those redirects would be nice. @tbekolay? I'll fix the URL anyway, though.